### DISCUSSION TITLE
Telegram Push fails when SITE_URL contains single quote

### DISCUSSION THREAD

**Discussion by ReiSuzunami** [walinejs/waline#1903] [Q&A]
My site's name contains a single quote and the push service errored with this message:

`Telegram Notification Failed:{"ok":false,"error_code":400,"description":"Bad Request: can't parse entities: Character '#' is reserved and must be escaped with the preceding '\\'"}`

Same problem appeared in #276 . Hope there would be a solution.

**Comment by lizheming** [comment]
https://github.com/orgs/walinejs/discussions?discussions_q=telegram
